resources/weapons/rockets/AGR-20 M282.yaml

```diff
@@ -0,0 +1,5 @@
+name: LAU-131 pod - 7 x 2.75\" Hydra, Laser Guided Rkts M282, MPP APKWS

```

Comment:
These don't have to match the in game name (which is the reason for the clsid mapping; the game names change somewhat frequently). You can name them something simpler like `LAU-131 7xMPP APKWS` if you prefer.

Reply:
I did not know that. That is a lot easier, thanks.